models/tokens/ethereum/tokens_ethereum_erc20.sql

```diff
@@ -21558,4 +21558,2969 @@ FROM (VALUES
         ,('0xfb272d3acfcbf3e0f92bd01ffbcde92575c371f9', 'JPOW', 9)
         ,('0xfca99bf40253563f619b5fbaed621021b2987a09', 'zjz', 18)
         ,('0xfe60fba03048effb4acf3f0088ec2f53d779d3bb', '3d3d', 18)
+                ,('0x235c8ee913d93c68d2902a8e0b5a643755705726', 'BAG', 18)
```

Comment:
v nitpicky, but can we fix the spacing here to align? maybe it'd happen each time the script ran?

Reply:
fixed.